src/Features/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="..\..\Directory.Build.targets"/>
+
+  <ItemGroup Condition="'$(IsShipping)' == 'true'">
```

Comment:
❔ Why only shipping projects? Seems like the tests have also been updated to account for this.

Reply:
Yes, I will try to revert.